config/locales/devise.views.it.yml

```diff
@@ -0,0 +1,141 @@
+it:
```

Comment:
Wow, Italian :)
I like Italian but do we really need this locale in our app?

Reply:
Oops, missed that file. Deleted.